helix-rest/src/test/java/org/apache/helix/rest/server/TestDefaultMonitoringMbeans.java

```diff
@@ -36,9 +37,14 @@
   // For entire testing environment, we could have 2 - 4 rest server during the testing. So we dont
   // know which REST server got the request and report number. So we have to loop all of them to
   // report data.
-  @Test
+
+  // This is unstable test because the getcluster MBean is even not there after our call
+  // and this is not critical for all existing logic. So disable it now.
+  // TODO: Make MBean can be stable queried.
```

Comment:
Did you see `cleanupJMXObjects()`? helix-core tests use this to ensure all MBeans are cleared. Could you try this in your helix-rest test base and see if things become more stable?

Reply:
Tried use same logic of cleanupJMXObjects(), it does not work. The unstable part is the default metrics sometimes not showing up in MBeamServer. Need take a look later.